src/Commands/Issue.php

```diff
@@ -95,9 +91,9 @@ private function doExecute(Manager $args): Generator {
 
                 file_put_contents("{$path}/{$token}", $payload);
                 chown("{$path}/{$token}", $userInfo["uid"]);
-                chmod("{$path}/{$token}", 0660);
+                chmod("{$path}/{$token}", 0664);
```

Comment:
Yes, the challenge can be publicly readable. There's a `user` option, but that works only if you're running it as `root`.